packages/gator-permissions-snap/src/core/permissionHandlerContent.tsx

```diff
@@ -54,7 +69,22 @@ export const PermissionHandlerContent = ({
   tokenSymbol,
   tokenIconData,
   isJustificationCollapsed,
+  context,
+  tokenBalance,
+  tokenBalanceFiat,
 }: PermissionHandlerContentProps): SnapElement => {
+  const tokenBalanceComponent = tokenBalance ? (
+    <Text>{tokenBalance} available</Text>
```

Comment:
I think we need to do some rounding up here for `tokenBalance`. 

<img width="394" height="612" alt="Screenshot 2025-08-05 at 11 44 39" src="https://github.com/user-attachments/assets/2feef2d8-93af-4d64-b9ec-2a86552803fa" />